docs/metrics/extending-the-sdk/README.md

```diff
@@ -72,12 +71,88 @@ to the `MeterProvider` as shown in the example [here](./Program.cs).
 
 Not supported.
 
-## ExemplarFilter
+## ExemplarReservoir
 
-Not supported.
+> [!NOTE]
+> `ExemplarReservoir` is an experimental API only available in pre-release
+  builds. For details see:
+  [OTEL1004](../../diagnostics/experimental-apis/OTEL1004.md).
 
-## ExemplarReservoir
+Custom [ExemplarReservoir](../customizing-the-sdk/README.md#exemplarreservoir)s
+can be implemented to control how `Exemplar`s are recorded for a metric:
 
-Not supported.
+* `ExemplarReservoir`s should derive from `FixedSizeExemplarReservoir` (which
+  belongs to the [OpenTelemetry](../../../src/OpenTelemetry/README.md) package)
+  and implement the `Offer` methods.
+* The `FixedSizeExemplarReservoir` constructor accepts a `capacity` parameter to
+  control the number of `Exemplar`s which may be recorded by the
+  `ExemplarReservoir`.
+* The `virtual` `OnCollected` method is called after the `ExemplarReservoir`
+  collection operation has completed and may be used to implement cleanup or
+  reset logic.
+* The `bool` `ResetOnCollect` property on `ExemplarReservoir` is set to `true`
+  when delta aggregation temporality is used for the metric using the
+  `ExemplarReservoir`.
+
+```csharp
```

Comment:
describe a bit about what this example reservoir does.

Reply:
Added